### PR TITLE
CLI rollup

### PR DESCRIPTION
**Pull request by tamird**
Closes #2102.
Closes #2156.

### CODE REVIEW

File: cli/kv.go [hunk 1]
```diff
@@ -119,23 +115,19 @@ func runPut(cmd *cobra.Command, args []string) {
...
-	keys := make([]string, 0, count)
-	values := make([]string, 0, count)
+	var b client.Batch
 	for i := 0; i < len(args); i += 2 {
-		keys = append(keys, unquoteArg(args[i], true /* disallow system keys */))
-		values = append(values, unquoteArg(args[i+1], false))
+		b.Put(
+			unquoteArg(args[i], true /* disallow system keys */),
+			unquoteArg(args[i+1], false),
+		)
```
Comment (petermattis) [inline cli/kv.go]: That's some unusual formatting. Any reason to put first arg on a newline and the closing paren on a line by itself?

File: cli/kv.go [hunk 2]
```diff
@@ -143,7 +135,7 @@ func runPut(cmd *cobra.Command, args []string) {
 	}
 }
 
-// A incCmd command increments the value for one or more keys.
+// A incCmd increments the value for one or more keys.
```
Comment (petermattis) [inline cli/kv.go]: `s/A/an/g`

File: cli/kv.go (hunk 1 above)
Comment (tamird) [inline cli/kv.go]: symmetry; is there precedent for another style?

**Comment by petermattis** [comment]
LGTM

File: cli/kv.go (hunk 1 above)
Comment (petermattis) [inline cli/kv.go]: Symmetry with what? I haven't seen this style used elsewhere.

**Pull request merged**